base/docs/helpdb/Base.jl

```diff
@@ -4204,11 +4204,9 @@ rand
 """
     base(base, n, [pad])
 
-Convert an integer to a string in the given base, optionally specifying a number of digits
-to pad to. The base can be specified as either an integer, or as a `UInt8` array of
-character values to use as digit symbols.
+Convert an integer to a string in the given base, optionally specifying a number of digits to pad to.
 """
-base
+base(base, n, pad)
```

Comment:
I don't think you needed to add the signature, this may make the docstring not match when you ask about base with fewer than 3 inputs